package.json

```diff
@@ -53,6 +53,7 @@
     "jsdom": "^24.0.0",
     "lodash-es": "^4.17.21",
     "lottie-react": "^2.4.0",
+    "lucide-react": "^0.378.0",
```

Comment:
Do you really need a whole library for two icons? Can't you use the svgs individiually instead?

Reply:
Done